ipaserver/install/upgradeinstance.py

```diff
@@ -161,6 +163,29 @@ def __save_config(self):
             else:
                 self.backup_state('nsslapd-global-backend-lock', global_lock)
 
+        with open(self.filename, "r") as in_file:
+            parser = GetEntryFromLDIF(in_file, entries_dn=[COMPAT_DN])
+            parser.parse()
+
+            try:
+                compat_entry = parser.get_results()[COMPAT_DN]
+            except KeyError:
+                pass
```

Comment:
Just add a `return` here and remove the else below.

Reply:
Done.